### ISSUE TITLE
[feat]: Open JSON(B) in Vim/JQ

### ISSUE THREAD

**Issue by prochac** [jorgerojas26/lazysql#98]
Could the external editor feature be duplicated for column results?
In some cases, we use JSON/JSONB in Potgres. And the column content is sometimes big. It would be nice to be able to see it formatted. The `jq` filter is nice to have. (I guess should be possible in Vim, if I learn it, right?)

**Comment by jorgerojas26** [comment]
That's a good feature. I'm thinking about implementing it just like MongoDB Compass does it, having a breadcrumb of result tables, where you can just press "Enter" on a column and if it has a complex structure (like json), create and navigate a step further in the breadcrumb and show the data in a table.

**Comment by jorgerojas26** [comment]
Of course, the "Open in external editor" is nice as well and i plan to support more of that.
